### PR TITLE
MB-3661 Add DB comments to Roci tables

### PR DESCRIPTION
**Pull request by sandy-wright**
## Description

This PR adds database comments to the tables and columns identified for Team Roci. Comments were either taken directly from [this spreadsheet](https://docs.google.com/spreadsheets/d/1pQVZdi5ttQ67DIcBAgk9INUH9oVqTDpzL2ciMAJuBV8/edit#gid=0), the API YAML files, or from discussions on Slack. 

The tables assigned to Roci:
- `client_certs`
- `electronic_orders`
- `electronic_orders_revisions`
- `entitlements` **
- `move_documents`
- `moves` **
- `moving_expense_documents`
- `mto_agents` **
- `mto_service_item_customer_contacts`
- `mto_service_item_dimensions` **
- `mto_service_items` **
- `mto_shipments` **
- `webhook_notifications`
- `webhook_subscriptions`

** these tables were already commented and needed only a few additions or updates.

## Setup

Reset and migrate your database:

```sh
make db_dev_reset && make db_dev_migrate
```

Generate the schema interface:

```sh
make schemaspy
```

Then navigate to `./tmp/schemaspy` and locate the `index.html` file. Open this up in your browser and read the new comments 😄 

## Code Review Verification Steps

* Any new migrations/schema changes:
  * [x] Have been communicated to #g-database
* [x] Request review from a member of a different team.
* [x] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3661) for this change
* [MTO Database Field Definitions](https://docs.google.com/spreadsheets/d/1pQVZdi5ttQ67DIcBAgk9INUH9oVqTDpzL2ciMAJuBV8/edit#gid=0)
* [Database Documentation Proposal](https://docs.google.com/document/d/1odrsCfxi131FaN831v6b3eH1FNqMYCvPJUUbZno6bkA/edit#heading=h.smzrx57dszks)

### CODE REVIEW

**Review by shimonacarvalho — APPROVED** [review]
Omg that is a lot of typing! Awesome work and looks good to me but you need someone on db to approve, so deep or reggie.

File: migrations/app/schema/20200908162414_roci_db_comments.up.sql [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+COMMENT ON COLUMN client_certs.allow_prime IS 'Indicates whether or not the cert grants access to the Prime API';
+COMMENT ON COLUMN client_certs.created_at IS 'Date & time the client cert was created';
+COMMENT ON COLUMN client_certs.updated_at IS 'Date & time the client cert was last updated';
+
+-- electronic_orders
+COMMENT ON TABLE electronic_orders IS 'Represents the electronic move orders issued by a particular branch of the military';
+COMMENT ON COLUMN electronic_orders.orders_number IS 'A (generally) unique number identifying the orders, corresponding to the ORDERS number (Army), the CT SDN (Navy, Marines), the SPECIAL ORDER NO (Air Force), the Travel Order No. (Coast Guard), or the Travel Authorization Number (Civilian)';
+COMMENT ON COLUMN electronic_orders.edipi IS 'Electronic Data Interchange Personal Identifier, the 10 digit DoD ID Number of the service member';
+COMMENT ON COLUMN electronic_orders.issuer IS 'The organization that issued the orders (army, navy, etc.)';
+COMMENT ON COLUMN electronic_orders.created_at IS 'Date & time the electronic order was created';
```
Comment (LeDeep) [inline migrations/app/schema/20200908162414_roci_db_comments.up.sql:26]: some of the already merged comments use `Timestamp when the record was first created.` as the language for `created_at`. And similar for `updated_at`. For consistency maybe we should use one or the other, but not both?

This isn't for this PR, but maybe it's worth creating another ticket to update for consistency? cc @reggieriser

**Review by LeDeep — APPROVED** [review]
nice job! 🚀  one comment, non-blocking.

File: migrations/app/schema/20200908162414_roci_db_comments.up.sql (hunk 1 above)
Comment (sandy-wright) [reply migrations/app/schema/20200908162414_roci_db_comments.up.sql:26]: Yeah, I was wondering about this too! Picking one format for the timestamp fields would be nice

**Pull request merged**